moped-editor/src/components/GridTable/GridTableFilters.js

```diff
@@ -115,42 +115,54 @@ const GridTableFilters = ({
     filterState.filterParameters
   );
 
-  /**
-   * The default structure of an empty field
-   * @type {Object}
-   * @property {string} id - The uuid of the field
-   * @property {string} field - The name of the column
-   * @property {operator} operator - The name of the operator
-   * @property {string[]} availableOperators - A string array containing the names of available operators
-   * @property {string} gqlOperator - A string containing the GraphQL operator
-   * @property {string} envelope - The a pattern to use as an envelope
-   * @property {string} value - The text value to be searched
-   * @property {string} type - The type of field it is (string, number, etc.)
-   * @constant
-   * @default
-   */
-  const defaultNewFieldState = {
-    id: null,
-    field: null,
-    operator: null,
-    availableOperators: [],
-    gqlOperator: null,
-    envelope: null,
-    placeholder: null,
-    value: null,
-    type: null,
-    specialNullValue: null,
-    label: null,
-  };
-
   /**
    * Generates a copy of an empty field
    * @param uuid
    * @return {Object}
    */
-  const generateEmptyField = (uuid) => {
+  const generateEmptyField = useCallback((uuid) => {
```

Comment:
This `useCallback` works great and gets around the dependency array warnings that calling `generateEmptyField` inside of a `useEffect` probably caused. 

I think another way to get around this is pulling the definitions of `defaultNewFieldState` and `generateEmptyField` outside of the `GridTableFilters` component and into the top level of this file since they don't rely on any state. Just a different approach and no change needed here! There are many ways to handle this.